src/poliastro/plotting.py

```diff
@@ -198,6 +199,55 @@ def plot(self, orbit, label=None, color=None):
 
         return lines
 
+    def set_lines_up_to_instant(frame, self, x, y, lines):
+        """Plots a frame of the precomputed trajectory.
+
+        Parameters
+        ----------
+        frame : current plotting frame.
+        x : x values of the trajectory to plot.
+        y : y values of the trajectory to plot.
+        lines : plot information
+        """
+        lines.set_data(x[:int(frame)], y[:int(frame)])
+        return lines
+
+    def animate(self, orbit, label=None, color=None):
+        """Plots state and osculating animated orbit in their plane.
+
+        """
+        if not self._frame:
+            self.set_frame(*orbit.pqw())
+
+        if (orbit, label) not in self._orbits:
+            self._orbits.append((orbit, label))
+
+        self.set_attractor(orbit.attractor)
+        self._redraw_attractor(orbit.r_p * 0.15)  # Arbitrary Threshhold
+
+        _, positions_in_tuples = orbit.sample(self.num_points)
+        positions = positions_in_tuples.get_xyz().transpose()
+        x, y = self._project(positions)
+
+        # The following should be in a init_animation function, but I [aeronau] still don't know how to pass the arguments to the init function
```

Comment:
As far as I understand, the `init_func` has no arguments. https://matplotlib.org/api/_as_gen/matplotlib.animation.FuncAnimation.html I don't know whether there's a better way to do this, perhaps we should read some tutorial.